.github/workflows/Semnatic-PR.yml

```
@@ -0,0 +1,17 @@
+name: Lint PR
+'on':
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+permissions:
+  pull-requests: read
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
quotes shouldn't be needed